src/background/proxyHandler/proxyUtils.js

```diff
@@ -16,20 +16,44 @@ export const ProxyUtils = {
     return { type: "direct" };
   },
 
+  /**
+   * Checks if a given browser proxy setting is valid.
+   *
+   * @param {BrowserSettingValue} settingValue - The proxy settings object as defined by the
+   *   WebExtensions API.
+   * @returns {boolean} True if the setting is valid and contains a usable proxy configuration, false otherwise.
+   */
   browserProxySettingIsValid(settingValue) {
```

Comment:
This just checked if the strings are undefined. 
I got {mode: system, http:""} - which is not a valid proxy but valid for the old function